README.md

```diff
@@ -1,71 +1,214 @@
+
+Another potential avenue for using `spokestack` on Windows 10 is from [anaconda](https://www.anaconda.com/). This is without support for Text To Speech (TTS) though due to the Lame dependency. PortAudio, on the other hand, can be installed via `conda`.
+
+```shell
+conda install portaudio
+```
 
-Add voice capability to your projects with one line.
+### Installation with pip
 
-    pip install spokestack
+Once system dependencies have been satisfied, you can install the library with the following.
+
+```shell
+pip install spokestack
+```
 
 ## Development
 
 ### Setup
 
 We use `pyenv` for virtual environments. Below you will find the step-by-step commands to install a virtual environment.
 
-    pyenv install 3.8.6
-    pyenv virtualenv 3.8.6 spokestack
-    pyenv local spokestack
-    pip install -r requirements.txt
+```shell
+pyenv install 3.8.6
+pyenv virtualenv 3.8.6 spokestack
+pyenv local spokestack
+pip install -r requirements.txt
+```
+
+### Install Tensorflow
+
+This library requires a way to run [TFLite](https://www.tensorflow.org/lite) models. There are two primary ways to add this ability. The first, is installing the full [Tensorflow](https://www.tensorflow.org/) library. In most instances you should be able to install this with `pip`, but in the case of any installation issues follow the installation [guide](https://www.tensorflow.org/install).
+
+#### Tensorflow
+
+The full Tensorflow package is installed with the following:
+
+```shell
+pip install tensorflow
+```
+
+#### TFLite Interpreter (Embedded Devices)
+
+In use cases where you require a small footprint, such as on a raspberry pi or similar embedded devices, you will want to install the TFLite Interpreter. You can install it for your platform by following the instructions at [TFLite Interpreter](https://www.tensorflow.org/lite/guide/python#install_just_the_tensorflow_lite_interpreter).
+
+### Audio Input
+
+We offer a [PyAudio](https://people.csail.mit.edu/hubert/pyaudio/) interface for audio input. You can initialize it like this:
+
+```python
+from spokestack.io.pyaudio import PyAudioInput
+
+mic = PyAudioInput()
+mic.start()
+frame = mic.read()
+```
+
+### Voice Activity Detection (VAD)
+
+Voice activity detection listens to a frame of audio and determines if a voice is present. Our VAD uses [WebRTC](https://github.com/wiseman/py-webrtcvad). To initialize the VAD, add the following:
+
+```python
+from spokestack.vad.webrtc import VoiceActivityTrigger
+
+vad = VoiceActivityTrigger()
+```
+
+### Wakeword Detection
+
+Also known as keyword spotting (kws), wakeword is the way to get your applications attention with voice. This is normally through a specific keyword or phrase. The default wakeword for this library is "Spokestack". Wakeword can be added like this:
```

Comment:
It's the only model included in a spokestack account by default. it's the wakeword for all of the tutorials. it's the default wakeword. This seems like an arbitrary distinction.

Reply:
The difference is that the library provides a way to _run_ wakeword detection, not a wakeword itself. The library is code; the wakeword models are data. We give out one wakeword for free, so it's the one we use in the tutorials, but we don't distribute it with the libraries themselves, so the libraries don't _have_ a wakeword by default (you can't use a wakeword in your app without a separate download); they just have a component capable of using the models.

Like I said, it's a nitpick as far as readme wording goes (once you add the links to the wakeword models so that people don't assume it works out of the box); I'm just usually biased toward precision, sometimes overly so.

Reply:
It might be better to make it work out of the box.